assets/src/dashboard/app/views/editorSettings/adManagement/test/adManagement.js

```diff
@@ -47,6 +52,7 @@ describe('Editor Settings: Ad Management group settings <AdManagement />', funct
         publisherId=""
         adSenseSlotId=""
         adManagerSlotId=""
+        siteKitStatus={{ ...defaultsiteKitStatus }}
```

Comment:
nit: I don't think we need to create a new object here but it's not a big deal

Reply:
Agreed. Same for below.

Reply:
Copied from https://github.com/google/web-stories-wp/blob/f459efbc4a2c9546a0ca2fed1301e4f222fa2340/assets/src/dashboard/app/views/editorSettings/googleAnalytics/test/googleAnalytics.js#L52 . Something you wrote @samwhale 😉